repo.json

```diff
@@ -1,12 +1,2590 @@
 {
-    "repo_name": "O3DE Extras - Extras gems, projects, and templates for O3DE",
+    "repo_name": "O3DE Extras",
     "origin": "o3de-extras",
     "repo_uri": "https://raw.githubusercontent.com/o3de/o3de-extras/development",
     "summary": "A repository to hold various gems, projects, and templates for the O3DE engine.",
-    "additional_info": "See the README.md at the root of this repository for more information",
+    "additional_info": "Extras gems, projects, and templates for O3DE. See the README.md at the root of this repository for more information",
     "last_updated": "2022-09-27",
+    "$schemaVersion": "1.0.0",
+    "gems_data": [
+        {
+            "gem_name": "AzQtComponentsForPython",
```

Comment:
How did this get into O3DE-extras??

Reply:
Looks like it was added ~10 months ago: https://github.com/o3de/o3de-extras/tree/development/Gems/AzQtComponentsForPython